draft-ietf-rats-corim.md

```diff
@@ -1118,6 +1118,21 @@ are acceptable states.
 Integrity Registers can be used to model the PCRs in a TPM or vTPM, in which case the identifier is the register index, or other kinds of vendor-specific measured objects.
 
 
+##### Linear Privilege Level {#sec-comid-linear-privlevel}
+
+A Linear Privilege Level describes an operating privilege for the target environment.
+The convention is that 0 is the highest privilege, and higher numbers correspond to fewer privileges.
```

Comment:
```suggestion
When expressed in positive integer ranges the convention is that 0 is the highest privilege, and higher numbers correspond to fewer privileges. No negative number representation is allowed, in this setting.
```

Reply:
What setting? This is convention.

Reply:
I don't understand this comment. Replace what with convention? I'm trying to encode a range comparison that requires an internalized notion of order.